walmart/walmart.py

```diff
@@ -86,6 +90,7 @@ class Resource(object):
 
     def __init__(self, connection):
         self.connection = connection
+        self.boundary = '7312ccd96db94419bf1d97f2c54bbad1'
```

Comment:
class attribute would be much better

Reply:
1. Boundary should be a random string not hardcoded. may be use UUID4 instead.
2. No point of binding boundary with class or instance, it should only be generated where ever it is needed.